content/en/docs/kubernetes/collector/components.md

```diff
@@ -2,7 +2,9 @@
 title: Important Components for Kubernetes
 linkTitle: Components
 # prettier-ignore
-cSpell:ignore: containerd crio filelog gotime horizontalpodautoscalers iostream k8sattributes kubelet kubeletstats logtag replicasets replicationcontrollers resourcequotas statefulsets varlibdockercontainers varlogpods
+spelling: 
```

Comment:
I don't know how to fix the spelling anymore please help

Reply:
Run `npm run fix:dict`.
Then only add words to the single `cSpell:ignore` line.
Rerun `npm run fix:dict` after you've added words, and before you push.

Let me know if that helps.